models/Asana.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const asanaSchema = new Schema({
+  sanskrit_name: { type: String, required: true },
+  english_name: { type: String, required: true },
+  img_url: { type: String, required: true },
+  description: { type: String, required: true },
+  levels: { type: Array, required: true },
+  category: { type: String, required: true },
+  benefit: { type: String, required: true },
+  next: { type: Array, required: true },
+});
+
+const Asana =
+  mongoose.models.Asana || mongoose.model("Asana", asanaSchema, "asanas");
```

Comment:
You're right. Then it's all fine. I had an issue myself the other day where I had a model called "tlaRental" and mongoose converted all letters to lowercase ones, but my collection in the database was called "tlaRental". Mongoose couldn't find the collection because it was looking for a collection in all lowercase. That's why I mentioned this lowercase thing. But if it's working for you, all good :)